### DISCUSSION TITLE
Conditional formatting for 4k (BIG number) visualization chart

### DISCUSSION THREAD

**Discussion by alex1404** [apache/superset#22247] [Q&A / Help]
Hi all, 
Is there a way to add Conditional formatting in 4k charts.
![image](https://user-images.githubusercontent.com/12210275/204343486-d7d72d44-f056-4a7c-921f-90d5cc41378c.png)
For example if the value above 100 the value will colored red else green.
Thanks.

**Comment by kamalkeshavani-aiinside** [comment]
No, such feature is not available. But maybe can be achieved with Handlebars chart available in 2.x
